packages/asset-swapper/src/swap_quoter.ts

```diff
@@ -589,8 +588,8 @@ export class SwapQuoter {
 
         const calcOpts: CalculateSwapQuoteOpts = opts;
 
-        if (calcOpts.rfqt !== undefined && this._shouldEnableIndicativeRfqt(calcOpts.rfqt, marketOperation)) {
-            calcOpts.rfqt.quoteRequestor = this._quoteRequestor;
+        if (calcOpts.rfqt !== undefined) {
```

Comment:
@feuGeneA I've changed this to _always_ pass through the `QuoteRequestor` (even if it is not an indicative quote.  Seems harmless from the code I've read through but I wanted to confirm w/ you